deepchecks/nlp/utils/text_properties.py

```diff
@@ -605,40 +620,36 @@ class TextProperty(TypedDict):
     output_type: str
 
 
-DEFAULT_PROPERTIES: Tuple[TextProperty, ...] = (
-    {'name': 'Text Length', 'method': text_length, 'output_type': 'numeric'},
-    {'name': 'Average Word Length', 'method': average_word_length, 'output_type': 'numeric'},
-    {'name': 'Max Word Length', 'method': max_word_length, 'output_type': 'numeric'},
-    {'name': '% Special Characters', 'method': percentage_special_characters, 'output_type': 'numeric'},
-    {'name': 'Language', 'method': language, 'output_type': 'categorical'},
-    {'name': 'Sentiment', 'method': sentiment, 'output_type': 'numeric'},
-    {'name': 'Subjectivity', 'method': subjectivity, 'output_type': 'numeric'},
-    {'name': 'Average Words Per Sentence', 'method': average_words_per_sentence, 'output_type': 'numeric'},
-    {'name': 'Readability Score', 'method': readability_score, 'output_type': 'numeric'},
-    {'name': 'Lexical Density', 'method': lexical_density, 'output_type': 'numeric'},
-    {'name': 'Toxicity', 'method': toxicity, 'output_type': 'numeric'},
-    {'name': 'Fluency', 'method': fluency, 'output_type': 'numeric'},
-    {'name': 'Formality', 'method': formality, 'output_type': 'numeric'},
-    {'name': 'Unique Noun Count', 'method': unique_noun_count, 'output_type': 'numeric'},
-)
+DEFAULT_PROPERTIES: Tuple[TextProperty, ...] = \
+    (
+        {'name': 'Text Length', 'method': text_length, 'output_type': 'numeric'},
+        {'name': 'Average Word Length', 'method': average_word_length, 'output_type': 'numeric'},
+        {'name': 'Max Word Length', 'method': max_word_length, 'output_type': 'numeric'},
+        {'name': '% Special Characters', 'method': percentage_special_characters, 'output_type': 'numeric'},
+        {'name': 'Language', 'method': language, 'output_type': 'categorical'},
+        {'name': 'Sentiment', 'method': sentiment, 'output_type': 'numeric'},
+        {'name': 'Subjectivity', 'method': subjectivity, 'output_type': 'numeric'},
+        {'name': 'Average Words Per Sentence', 'method': average_words_per_sentence, 'output_type': 'numeric'},
+        {'name': 'Readability Score', 'method': readability_score, 'output_type': 'numeric'},
+        {'name': 'Lexical Density', 'method': lexical_density, 'output_type': 'numeric'},
+        {'name': 'Toxicity', 'method': toxicity, 'output_type': 'numeric'},
+        {'name': 'Fluency', 'method': fluency, 'output_type': 'numeric'},
+        {'name': 'Formality', 'method': formality, 'output_type': 'numeric'},
+        {'name': 'Unique Noun Count', 'method': unique_noun_count, 'output_type': 'numeric'},
+    )
 
-ALL_PROPERTIES: Tuple[TextProperty, ...] = (
```

Comment:
Not sure what is the change here

Reply:
It was indented in a way that annoyed me

Reply:
This specific line - moved the parenthesis to the next line so it would auto-indent correctly.